spec/models/iiif_resource_spec.rb

```diff
@@ -301,165 +301,166 @@
-          "viewingHint": "individuals",
-          "structures": [
-            {
-              "@id": "https://localhost.localdomain/collections/2b88qc199/manifest/range/g70272079000020",
-              "@type": "sc:Range",
-              "label": "Logical",
-              "viewingHint": "top"
-            }
-          ],
-          "metadata": [
-            {
-              "label": "Exhibit",
-              "value": ["princeton-best"]
-            }
-          ],
-          "manifests": [
-            {
-              "@id": "https://localhost.localdomain/scanned_resources/1r66j1149/manifest",
-              "@type": "sc:Manifest",
-              "label": ["Christopher and his kind, 1929-1939"],
-              "viewingHint": "individuals",
-              "viewingDirection": "left-to-right"
-            }
-          ]
-        }
-      end
-
-      let(:collections_manifest) do
-        {
-          "@context": "http://iiif.io/api/presentation/2/context.json",
-          "@id": "https://localhost.localdomain/collections/manifest",
-          "@type": "sc:Collection",
-          "label": "Figgy Collections",
-          "description": "All collections which are a part of Figgy.",
-          "viewingHint": "individuals",
-          "structures": [
-            {
-              "@id": "https://localhost.localdomain/collections/manifest/range/g70272257673280",
-              "@type": "sc:Range",
-              "label": "Logical",
-              "viewingHint": "top"
-            }
-          ],
-          "collections": [
-            {
-              "@id": "https://localhost.localdomain/collections/2b88qc199/manifest",
-              "@type": "sc:Collection",
-              "label": "princeton",
-              "viewingHint": "individuals",
-              "structures": [
-                {
-                  "@id": "https://localhost.localdomain/collections/2b88qc199/manifest/range/g70272079000020",
-                  "@type": "sc:Range",
-                  "label": "Logical",
-                  "viewingHint": "top"
-                }
-              ],
-              "metadata": [
-                {
-                  "label": "Exhibit",
-                  "value": ["princeton-best"]
-                }
-              ]
-            }
-          ]
-        }
-      end
-
-      before do
-        stub_request(
-          :get,
-          "https://localhost.localdomain/scanned_resources/1r66j1149/manifest"
-        ).to_return(body: JSON.generate(scanned_resource_manifest))
-        stub_request(
-          :get,
-          "https://localhost.localdomain/collections/2b88qc199/manifest"
-        ).to_return(body: JSON.generate(collection_manifest))
-        stub_request(
-          :get,
-          url
-        ).to_return(body: JSON.generate(collections_manifest))
-        stub_request(
-          :head,
-          url
-        ).to_return(headers: { 'Content-Type' => 'application/json; charset=utf-8' })
-      end
-    end
+    # TODO: Add tests
```

Comment:
Question: Why's all this commented?

Reply:
There's no test in that context and rubocop is complaining. I wasn't sure if the test was removed during this process with the idea that it needed to be put back eventually.

Reply:
Oh yeah, it was all set up to test that a pretty error happened when there was a bad collection metadata hash.  I ripped it out because it was a lot of code to just error a little differently. https://github.com/pulibrary/dpul/blob/main/spec/models/iiif_resource_spec.rb#L469 was the test. I think we can just delete this whole block.

Reply:
Done.